bodhi/server/templates/fragments.html

```diff
@@ -79,14 +79,12 @@
 % if comment.user.name == "bodhi":
 <div class="row">
         <div class="col-12 text-muted font-size-09">
-          <div class="card border-0">
-            <div class="card-body border-0 p-2">
-                <i class="fa fa-info-circle"></i>
-                <span class="font-weight-bold">${comment.text | n}</span>
-                <a href="${comment.url()}" class="notblue float-right" data-toggle="tooltip" title="${comment.timestamp.strftime("%Y-%m-%d %H:%M:%S")} (UTC)">
-                    ${util.age(comment.timestamp)}
-                  </a>
-            </div>
+          <div class="d-flex p-2 align-items-top">
+            <i class="fa fa-info-circle pt-2"></i>
+            <div class="font-weight-bold">${util.markup(comment.text) | n}</div>
```

Comment:
In #3764 I've added a new option to util.markup() to not use the Bodhi markup extensions. I think we should use that option here to avoid update links already posted in markup form to be modified by the new extension.

Reply:
roger that! we will have to merge this one after yours goes in though. So will add WIP.

Reply:
PR #3764 is merged, you can adjust this now if need be.